admin/base/urls.py

```diff
@@ -9,7 +9,7 @@
     ### ADMIN ###
     url(base_pattern,
         include(patterns('',
-                         url(r'^$', views.root),
+                         url(r'^$', views.root, name='home'),
```

Comment:
Not sure if this is a good naming schema, specifically `views.root`, we should definitely discuss this.